Find a functionally equivalent code:android.widget.DatePicker.getMaxDate:COMMENT
Method Modifier: public      
Comment:/**
 * Gets the maximal date supported by this {@link DatePicker} in
 * milliseconds since January 1, 1970 00:00:00 in
 * {@link TimeZone#getDefault()} time zone.
 * <p>
 * Note: The default maximal date is 12/31/2100.
 * <p>
 *
 * @return The maximal supported date.
 */

@@ -1,3 +1,3 @@
 {
-    return mCalendarView.getMaxDate();
+    return mDelegate.getMaxDate().getTimeInMillis();
 }

